hack/add-license.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+rerun_to_add="Have checked out files without copyright, if you want to add them, please rerun 'make copy-copyright add=true'"
+
+function print_rerun_intro {
+    echo "$rerun_to_add"
```

Comment:
I think you can simply add the line here directly without the variable `rerun_to_add`, I didn't see it's being used anywhere else.